apps/web/src/pages/[document].tsx

```diff
@@ -152,8 +161,34 @@ const DocumentPage: NextPage<Props> = ({ page: initialPage }) => {
     content: page?.data ?? [],
   };
 
+  const metaTags = formatMetaTags({ page });
+
   return (
     <div>
+      <Head>
+        <meta
+          name="keywords"
+          content="web3, web3 document, web3 knowledge management, web3 data, web3 analytics, web3 documents, web3 onchain data, web3 sharing, web3 content, blockchain analytics, blockchain writing"
+        />
+        <meta name="robots" content="index, follow" />
+        <title>{metaTags.title}</title>
+
+        {/* open graph */}
+
+        <meta property="og:title" content={metaTags.title} />
+        <meta property="og:description" content={metaTags.description} />
+        <meta property="og:type" content="website" />
```

Comment:
We could use `<meta property="og:type" content="article" />` for documents/pages. It can enrich meta data.

<img width="573" alt="image" src="https://user-images.githubusercontent.com/3134895/232283210-a95dc1fd-2198-4e66-bae1-c8cb7932b62b.png">